src/main/java/com/quirko/logic/bricks/RandomBrickGenerator.java

```diff
@@ -21,6 +23,29 @@ public RandomBrickGenerator() {
         brickList.add(new SBrick());
         brickList.add(new TBrick());
         brickList.add(new ZBrick());
+        if(diff != DifficultyType.EASY){//extra hard shape for increase difficulty
+            brickList.add(new SBrick());
+            brickList.add(new TBrick());
+            brickList.add(new ZBrick());
+        }
+         if(diff != DifficultyType.HARD){//extra easy shape for decrease difficulty
+            brickList.add(new IBrick());

```

Comment:
Bu bölgedeki kod şu anda çok karmaşık. bu if ler bloğunu daha anlaşılır hale getir.

Örneğin block classlarını kolay orta zor diye guruplara böl ve bunları kolay orta zor isimli arraylere at. Daha sonra zorluk durumuna göre bu arraylerdeki elemanları addAll yaparak brickList'e ekle.